### PR TITLE
Fix crash in page scraper

### PR DESCRIPTION
**Pull request by pisoiasul**

### CODE REVIEW

File: services/facebook.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def get_liked_pages(user_id, access_token):
         logger.info(json.dumps(liked_pages, indent=2))
         logger.info("Found {} in this batch".format(len(liked_pages.get('data', []))))
         all_liked_pages += liked_pages.get('data', [])
-        nxt = liked_pages['paging'].get('next')
+        nxt = liked_pages['paging'].get('next') if 'paging' in liked_pages else None
```
Comment (akellehe) [inline services/facebook.py]: If you change `['paging']` to `.get('paging')` it will return `None` when `'paging'` doesn't exist

Comment (pisoiasul) [inline services/facebook.py]: Ok let me amend that

Comment (pisoiasul) [inline services/facebook.py]: Won't it crash at the next .get('next') then ?

Comment (pisoiasul) [inline services/facebook.py]: Done

File: services/facebook.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def get_liked_pages(user_id, access_token):
         logger.info(json.dumps(liked_pages, indent=2))
         logger.info("Found {} in this batch".format(len(liked_pages.get('data', []))))
         all_liked_pages += liked_pages.get('data', [])
-        nxt = liked_pages['paging'].get('next')
+        nxt = liked_pages.get('paging').get('next')
```
Comment (akellehe) [inline services/facebook.py]: Oh, sorry @pisoiasul, there's one more detail... If `.get('paging')` returns `None` you'll get an `AttributeError`. The second argument to `get()` is the _default_ return value for when the attribute doesn't exist. You should pass a `{}` there so the next call to `get()` on that return value just returns `None`... e.g.
```python
nxt = liked_pages.get('paging', {}).get('next')
```

Comment (pisoiasul) [inline services/facebook.py]: Done

**Pull request merged**